app/controllers/comments_controller.rb

```diff
@@ -78,17 +78,16 @@ def destroy
     respond_with(@comment)
   end
 
-protected
+  protected
 
   def comment_params
     params[:comment].permit!
   end
 
-private
+  private
 
   #----------------------------------------------------------------------------
   def extract_commentable_name(params)
-    params.keys.detect {|x| x =~ /_id$/ }.try(:sub, /_id$/, '')
+    params.keys.detect { |x| x =~ /_id$/ }.try(:sub, /_id$/, '')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.